Sources/Slacket/Parsers/ParserType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParserType.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+import Kitura
+
+// To define models that can be parsed
+protocol Parsable {}
+
+// To define parsers
+protocol ParserType {}
+
+protocol ParserDecoder: ParserType {
+    
+    func parse(body: ParsedBody) -> Parsable
```

Comment:
I'm not sure how it will work. For example, let's think about `SlackCommand` model. Then we'd like to have some parser/decoder: `SlackCommandDecoder`. But imo we'd like it returns `SlackCommandDecoder`, not some `Parsable` object. Am I understand it right?